src/style.css

```diff
@@ -1 +1,138 @@
-/* add styles here */
+/* #region fonts */
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+/* #endregion fonts */
+
+/* #region normalize */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* #endregion normalize */
+
+/* #region page */
+.page {
+  font-style: Avenir;
+  font-weight: 300;
+  padding: 20px 8px 0;
+}
+
+/* #endregion page */
+
+/* #region container (Is used as a DOM element) */
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 100%;
+  padding: 0;
+}
+
+/* #endregion container */
+
+/* #region form */
+.form {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+.form__search-button--big {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.form__search-button--small {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+}
+
+/* #endregion form */
+
+/* #region icon-search */
+.search-button {
+  margin: 0;
+  padding: 0;
+  border: none;
+  background-color: transparent;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+  z-index: 1;
+}
+
+.search-button--size--big {
+  width: 19px;
+  height: 19px;
+}
+
+.search-button--size-small {
+  width: 11px;
+  height: 11px;
+}
+
+/* #endregion icon-search */
+
+/* #region input-search */
+.input-search {
+  width: 100%;
+  border: 1px solid #E1E7ED;
+  border-radius: 3px;
+  box-shadow: 0 1px 8px #3D4E611A;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
+
+.input-search:hover {
+  box-shadow: 0 3px 8px #3D4E6133;
+}
+
+.input-search:focus {
+  background: linear-gradient(180deg, #fff 0%, #F6F6F7 100%);
+  border: 1px solid #E1E7ED;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.input-search--size--big {
+  height: 70px;
+  text-indent: 62px;
+  font-size: 16px;
+}
+
+.input-search--size--big::placeholder {
+  font-weight: 300;
+  color: #3D4E61;
+}
+
+.input-search--size--big:focus {
+  color: #3D4E61;
+  text-shadow: 0 4px 4px #00000040;
+}
+
+.input-search--size--small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 19px;
+}
+
+.input-search--size--small::placeholder {
+  font-weight: 300;
+  color:#3D4E61;
+}
```

Comment:
The styles for `input-search--size--big::placeholder` and `input-search--size--small::placeholder` are the same, which means with either modifier the input-seach block will have the same style; since input-search is always used with one of these modifiers maybe you should consider not making this style belonging to the modifier but to the actual input-search; that is: 

```css
.input-search::placeholder {
  font-weight: 300;
  color: #3D4E61;
}
```